packages/core/src/hooks/useLocalStorage.ts

```diff
@@ -8,15 +8,15 @@ function getValueFromLocalStorage<T>(key: string, defaultValue: T) {
     // Get from local storage by key
```

Comment:
I think outer `try` block here is no longer necessary.

Reply:
thanks, removed